tests/unit/amo/components/TestReportUserAbuse.js

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+
+import ReportUserAbuse, {
+  ReportUserAbuseBase,
+} from 'amo/components/ReportUserAbuse';
+import { setError } from 'core/actions/errors';
+import {
+  hideUserAbuseReportUI,
+  loadUserAbuseReport,
+  sendUserAbuseReport,
+  showUserAbuseReportUI,
+} from 'amo/reducers/userAbuseReports';
+import DismissibleTextForm from 'ui/components/DismissibleTextForm';
+import ErrorList from 'ui/components/ErrorList';
+import { dispatchClientMetadata } from 'tests/unit/amo/helpers';
+import {
+  createFakeUserAbuseReport,
+  createStubErrorHandler,
+  createUserAccountResponse,
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+
+
+describe(__filename, () => {
+  const defaultRenderProps = {
+    errorHandler: createStubErrorHandler(),
+    i18n: fakeI18n(),
+    store: dispatchClientMetadata().store,
+    user: createUserAccountResponse({ id: 9 }),
+  };
+
+  function renderShallow({ ...props } = {}) {
+    return shallowUntilTarget(
+      <ReportUserAbuse {...defaultRenderProps} {...props} />,
```

Comment:
I don't think this is a good idea to use a static object here (`defaultRenderProps`) because it means all tests share one `store` object. You should either create `store` in `beforeEach()` or create it each time `renderShallow()` is called (by calling a function or something). The `errorHandler` prop would have the same problem.

Reply:
Agreed. These tests are copypasta from the add-on user abuse reports, but I'll tidy at least these ones up.